### ISSUE TITLE
`statusMessage` not included in the list of intrinsic attributes returned by `api/v2/search/tags`

### ISSUE THREAD

**Issue by fabrizio-grafana** [grafana/tempo#3081]
**Describe the bug**
The `api/v2/search/tags` endpoint (and probably also the associated v1 version) does not include the `statusMessage` intrinsic attribute, added with [this PR](https://github.com/grafana/tempo/pull/2848) more than one month ago, in the list of tags. Partial example of response:
```
{"name":"intrinsic","tags":["duration","kind","name","rootName","rootServiceName","status","traceDuration"]}
```

**To Reproduce**
1. Start Tempo locally, latest version
2. Call the `api/v2/search/tags` endpoint (e.g., `http://localhost:3200/api/v2/search/tags`), for example from browser

**Expected behavior**
Partial output:
```
{"name":"intrinsic","tags":[...,"status","statusMessage", ...]}
```
In other words, similarly for instance to `status`, I would expect `statusMessage` to be included too in this list.

**Additional Context**
I tried to have a look by myself at Tempo's code, but I didn't manage to understand whether this is actually a real bug and in such case where is the error.
